src/components/nc-slideshow.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <div class="nc-slideshow">
+    <div class="nc-slideshow__content" :style="{'width': `${width}px`}">
+      <ul class="list" v-touch:swipe="goToSlide"
+        :style="{
+          width: `${width * slideLength}px`,
+          transform: `translate3d(-${slideIndex * width}px, 0px, 0px)`,
+          transition: '-webkit-transform 500ms ease'
+        }">
+        <slot></slot>
+      </ul>
+    </div>
+    <div class="nc-slideshow__pagination">
+      <ul class="dots" aria-label="Pagination Slideshow" role="navigation">
+        <li 
+          :style="paginationStyle" 
+          v-for="(i, index) in slideLength" 
+          :key=index @click.stop="selectNewSlide(index)" 
+          :class="[ slideIndex === index ? paginationActiveClass : '']" 
+          role="tab"
+          :aria-label="ariaObject.itemIndex + (index + 1)"
+          :aria-controls="ariaObject.tabPanel + (index + 1)"
+          :aria-selected="slideIndex === index ? true : false"
+          :tabindex="slideIndex === index ? -1 : 0"
+        >
+          <span>{{ ariaText.ariaTextDots + index + 1 }}</span>
```

Comment:
este span debería tener una clase asignada a la que se le apliquen los estilos en ligar de al elemento directamente
```suggestion
          <span>{{ ariaText.ariaTextDots + index + 1 }}</span>
```

Reply:
He eliminado el span, poniendo en su lugar un button, añado la clase al button